src/main/java/bowling/domain/frame/FinalFrame.java

```diff
@@ -0,0 +1,96 @@
+package bowling.domain.frame;
+
+import bowling.domain.state.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static bowling.Constants.*;
+
+public class FinalFrame implements Frame {
+    private static final int FINAL_FRAME_NO = 10;
+
+    private State state;
+    private final StateHistory stateHistory;
+
+    private FinalFrame(State state, StateHistory stateHistory) {
+        this.state = state;
+        this.stateHistory = stateHistory;
+    }
+
+    public static Frame create() {
+        return new FinalFrame(new Ready(), new StateHistory());
+    }
+
+    public void play(int felledPin) {
+        assertFelledPin(felledPin);
+        state = state.play(felledPin);
+        stateHistory.add(state);
+    }
+
+    @Override
+    public State getState() {
+        return state;
+    }
+
+    @Override
+    public Frame getNext() {
+        return null;
+    }
+
+    @Override
+    public int getFrameNumber() {
+        return FINAL_FRAME_NO;
+    }
+
+    @Override
+    public boolean isEndedFrame() {
+        if(getBowledCount() == 1) {
+            return isEndedFrameBowledOnce();
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
            return false;
```

Reply:
앗 무조건 false 인 로직이군요!  감사합니다.